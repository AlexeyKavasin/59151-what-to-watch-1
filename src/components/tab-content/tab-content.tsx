import * as React from "react";
import {ITabContent} from "../../interfaces";

export class TabContent extends React.Component<ITabContent, null> {
    render() {
        const {activeTab, tabId, children} = this.props;
        return <React.Fragment>{activeTab === tabId ? children : null}</React.Fragment>
    }
}
