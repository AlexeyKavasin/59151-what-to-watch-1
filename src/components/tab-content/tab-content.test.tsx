import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {TabContent} from './tab-content';

describe(`TabContent snapshot tests`, () => {
  it(`TabContent renders correctly`, () => {
    const tree = renderer
      .create(
      <TabContent tabId={1} activeTab={1}/>)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
