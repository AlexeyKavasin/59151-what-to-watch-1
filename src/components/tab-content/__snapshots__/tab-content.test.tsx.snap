// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabContent snapshot tests TabContent renders correctly 1`] = `null`;
